.github/workflows/ci-build.yml

```diff
@@ -61,6 +61,30 @@ jobs:
     - name: Test
       run: cd build && make -j2 test
 
+  docs:
+    name: Docs
+
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends \
+          doxygen \
+          # EOF
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: CMake
+      run: mkdir build && cd build && cmake .. -DOPTION_DEDICATED=ON
+
+    - name: Build
+      run: cd build && make docs
+
```

Comment:
I did some thinking about this .. if we do not plan to fix the warnings for any of the three targets, running it in the CI is just a waste of CPU honestly :P So I guess we have to pick either one or the other .. either we plan to fix warnings in `ai-api` and `gs-api`, or we should not run it in the CI :) What do you think?

Reply:
I agree, a nightly run should be enough.